packages/react/src/components/Header/Header.jsx

```diff
@@ -11,42 +11,11 @@ import { Switcher20 } from '@carbon/icons-react';
 import { settings } from '../../constants/Settings';
 
 import HeaderActionGroup from './HeaderActionGroup';
+import { HeaderActionItemPropTypes, HeaderPanelPropTypes } from './HeaderPropTypes';
```

Comment:
I see we are deleting three objects but only importing two here. Is this on purpose?

Reply:
Yeah, ChildContentPropTypes is only used within HeaderActionItemPropTypes, so it didn't need to be re-imported.